lib/class-wp-json-comments-controller.php

```diff
@@ -191,6 +167,115 @@ public function delete_item( $request ) {
 		return array( 'message' => __( 'Deleted comment' ) );
 	}
 
+
+	/**
+	 * Check if a given request has access to read comments
+	 * 
+	 * @param  WP_JSON_Request $request Full details about the request.
+	 * @return bool
+	 */
+	public function get_items_permissions_check( $request ) {
+
+		// If the post id isn't specified, presume we can create
```

Comment:
@rachelbaker I'm not sure the best way to handle this, as `post_id` is a required param, this will never return `true` however, this is also used for the `OPTIONS` pre flight check, so if someone does `OPTIONS /wp/comments` do we say they have permission to `POST` there or not? (as opposed to `OPTIONS /wp/comments?post_id=123`